tsup.config.ts

```diff
@@ -0,0 +1,35 @@
+import { defineConfig } from 'tsup'
+import externalGlobal from 'esbuild-plugin-external-global'
+import { Plugin } from 'esbuild'
+
+export default defineConfig((options) => {
+  const esbuildPlugins: Plugin[] = []
+
+  if (options.platform === 'browser') {
+    esbuildPlugins.push(
+      externalGlobal.externalGlobalPlugin({
+        crypto: '{ webcrypto: window.crypto }',
+      })
+    )
+  }
```

Comment:
i need to add comment here about what this is doing. it's essentially replacing any imports of `crypto` with an object.

so this
```
import { webcrypto } from "crypto";
```
essentially becomes this
```
const { webcrypto } = { webcrypto: window.crypto };
```